tests/e2e/e2e_test.go

```diff
@@ -19,3 +26,200 @@ func (s *IntegrationTestSuite) TestUpgrade() {
 	}
 	s.T().Logf("SUCCESS")
 }
+
+func (s *IntegrationTestSuite) TestCLITxs() {
+	mainBranch := upgrade.VersionConfig{
+		ImageTag:  "main",
+		ImageName: "evmos",
+	}
+
+	s.runInitialNode(mainBranch, repoDockerFile)
+
+	testCases := []struct {
+		name      string
+		cmd       func() (string, error)
+		expPass   bool
+		expErrMsg string
+	}{
+		{
+			name: "fail - submit upgrade proposal, no fees & insufficient gas",
+			cmd: func() (string, error) {
+				return s.upgradeManager.CreateSubmitProposalExec(
+					"v11.0.0",
+					s.upgradeParams.ChainID,
+					5000,
+					true,
+					"--fees=",
+					"--gas=50000",
+				)
+			},
+			expPass:   false,
+			expErrMsg: "insufficient fee", // TODO when the PR https://github.com/evmos/evmos/pull/1386 is merged, this may fail with out of gas error
```

Comment:
this should have an error saying "please also provide a value for gas-prices or define fees='auto'"